renderer/main.js

```diff
@@ -314,6 +314,8 @@ function dispatch (action, ...args) {
     saveStateThrottled()
   }
   if (action === 'saveState') {
+    state.saved.window = state.saved.window || {}
+    state.saved.window.bounds = args[0]
```

Comment:
To avoid dispatching bounds as an argument we could do something like this instead:
```
{ x: window.screenX, y: window.screenY, width: window.outerWidth, height: window.outerHeight }
```
Thoughts?